From 073c45501001c2d1daa395dc46d170f417605f8a Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Mon, 19 Apr 2021 12:15:18 +0200
Subject: [PATCH 0066/1141] arm64: defconfig: add DYNAMIC_DEBUG in addons
 fragment

As in multi_v7 addons fragment, add DYNAMIC_DEBUG also in arm64 defconfig
addons fragment.

Change-Id: Ia0dd6d480d535826bff65ae050cfbd5686efeb4d
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/202236
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Alexandre TORGUE <alexandre.torgue@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -1,4 +1,5 @@
 #  ALPHABETICALLY SORTED: $LC_ALL=C sort fragment-02-defconfig-addons.config -o fragment-02-defconfig-addons.config
 CONFIG_ARCH_STM32=y
+CONFIG_DYNAMIC_DEBUG=y
 CONFIG_SERIAL_STM32=y
 CONFIG_SERIAL_STM32_CONSOLE=y
